api/migrations/versions/794506c08e10_.py

```diff
@@ -0,0 +1,38 @@
+"""empty message
+
+Revision ID: 794506c08e10
+Revises: 1afd8dfc8d05
+Create Date: 2021-08-16 11:37:15.501414
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '794506c08e10'
+down_revision = '1afd8dfc8d05'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('review_score', sa.Column('is_active', sa.Boolean, nullable=True))
+    op.add_column('guest_registration_answer', sa.Column('is_active', sa.Boolean(), nullable=True))
+    op.add_column('answer', sa.Column('is_active', sa.Boolean(), nullable=True))
```

Comment:
Can we also please update the accompanying `GuestRegistrationAnswer` model in `./api/app/invitedGuest/models.py` and `ReviewScore` model in `./api/app/reviews/models.py` and their accompanying references similar to what you did for `Answer` as well?

Reply:
Hi @amritpurshotam , I thought it would be better for me to submit these changes in 2 separate PRs since putting them here might make this PR too big and hard to review. What do you think?

Reply:
Hey @bdhobare, that's a good point but since these are quite similar and tightly coupled components I don't feel it would be more difficult on me to review. Perhaps if it helps you to develop against and keep it small on your side, maybe branch off this one, make the changes there, then merge back here once it's ready for review?

Reply:
@amritpurshotam please see the changes.